sanity-io/structured-content-2022
# Use data from Studio for primary navigation

# Use data from Studio for primary navigation

## Intent

Replaces the hardcoded items in _Nav.tsx_  with items fetched from Studio.
Solves [this](https://app.shortcut.com/sanity-io/story/16033/use-data-from-studio-primary-navigation-as-header-nav) Shortcut Story.

## Testing this PR

Step-by-step instructions which reviewers can use to verify that this PR
achieves its aim. Specify any prerequisites and assumptions. Be explicit.

1. Open the [Preview Deploy link](https://structured-content-2022-web-git-use-data-from-studio-pri-7f9396.sanity.build/) for this branch
2. Scroll down past the NavBlock to the point where the primary nav appears
3. Verify that the rendered items correspond with the data available in Studio
4. (optional) Modify the Primary navigation items and verify upon rebuilding the site that the updated links render in accordance with the Studio the changes you made
